### ISSUE TITLE
py38 task is broken on Travis

### ISSUE THREAD

**Issue by atugushev** [jazzband/pip-tools#858]
```
$ tox
GLOB sdist-make: /home/travis/build/jazzband/pip-tools/setup.py
py38-piplatest-coverage create: /home/travis/build/jazzband/pip-tools/.tox/py38-piplatest-coverage
py38-piplatest-coverage installdeps: pip, mock, pytest, pytest-cov
py38-piplatest-coverage inst: /home/travis/build/jazzband/pip-tools/.tox/.tmp/package/1/pip-tools-4.0.0.zip
py38-piplatest-coverage installed: atomicwrites==1.3.0,attrs==19.1.0,Click==7.0,coverage==4.5.3,importlib-metadata==0.18,mock==3.0.5,more-itertools==7.2.0,packaging==19.0,pip-tools==4.0.0,pluggy==0.12.0,py==1.8.0,pyparsing==2.4.1.1,pytest==5.0.1,pytest-cov==2.7.1,six==1.12.0,wcwidth==0.1.7,zipp==0.5.2
py38-piplatest-coverage run-test-pre: PYTHONHASHSEED='518900656'
py38-piplatest-coverage run-test: commands[0] | pytest
ERROR: InvocationError for command /home/travis/build/jazzband/pip-tools/.tox/py38-piplatest-coverage/bin/pytest (exited with code -11)
___________________________________ summary ____________________________________
ERROR:   py38-piplatest-coverage: commands failed
The command "tox" exited with 1.
```

See full log https://travis-ci.org/jazzband/pip-tools/jobs/563357768

**Comment by atugushev** [comment]
Clearing cache on Travis fixes the problem.

I've also found that PIP=latest doesn't install the latest pip. I've added to the tox command `pip --version` before `pytest` and made sure of that. See this log https://travis-ci.org/atugushev/pip-tools/jobs/563767730. Looks like tox installs last cached pip version. As a workaround i've added -U to `install_command= python -m pip install -U {opts} {packages}` and now it installs correct version.

**Comment by atugushev** [comment]
Fixen by #867

**Comment by atugushev** [comment]
> I've also found that PIP=latest doesn't install the latest pip. I've added to the tox command `pip --version` before `pytest` and made sure of that. See this log https://travis-ci.org/atugushev/pip-tools/jobs/563767730. Looks like tox installs last cached pip version. As a workaround i've added -U to `install_command= python -m pip install -U {opts} {packages}` and now it installs correct version.

FTR, this should be fixed in #868.
